packages/react/src/components/Storage/FileUploader/Tracker/index.tsx

```diff
@@ -54,106 +54,86 @@ export function Tracker({
   };
 
   return (
-    <Card
-      variation="outlined"
-      padding="0"
-      className="amplify-fileuploader-file"
-    >
-      <Flex direction="row" padding="xs medium" gap="small" alignItems="center">
-        <View className="amplify-fileuploader__img">{icon}</View>
-        {isEditing ? (
-          <Flex direction="row" flex="1" gap="small" alignItems="center">
-            <View flex="1">
-              <TextField
-                maxLength={1024}
-                width="100%"
-                label="file name"
-                size="small"
-                variation="quiet"
-                labelHidden
-                onChange={onChange}
-                value={name}
-              />
-            </View>
-            <Button size="small" variation="primary" onClick={onSaveEdit}>
-              Save
+    <View className={ComponentClassNames.FileUploaderFile}>
+      <View className={ComponentClassNames.FileUploaderFileImage}>{icon}</View>
+      {isEditing ? (
+        <Flex direction="row" flex="1" gap="small" alignItems="center">
```

Comment:
What do you think if we break elements in the logic down to smaller components? so it would be something like `is Editing? <EditView /> : <StartEditView />`